packages/lesswrong/components/sunshineDashboard/ModeratorUserInfo/RecentlyActiveUsers.tsx

```diff
@@ -0,0 +1,321 @@
+      const timeB = new Date(b.lastNotificationsCheck ?? 0).getTime();
+      return timeA - timeB
+    })
+  };
+
+  let sortedUsers = results;
+  switch (sorting) {
+    case "karma":
+      sortedUsers = usersSortByKarma(results);
+      break;
+    case "last20Karma":
+      sortedUsers = usersSortedByRecentKarma(results);
+      break;
+    case "downvoters":
+      sortedUsers = usersSortByDownvoters(results);
+      break;
+    case "lastMonthKarma":
+      sortedUsers = usersSortByLastMonthKarma(results);
+      break
+    case "userSortByRateLimitCount":
+      sortedUsers = userSortByRateLimitCount(results);
+      break;
+    case "lastNotificationsCheck":
+      sortedUsers = userSortByLastNotificationsCheck(results);
+      break
+  }
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.topBar}>
+        <Link to="/admin/moderation?view=sunshineNewUsers" className={classes.tabButton}>
+          Unreviewed Users
+        </Link>
+        <Link to="/admin/moderation?view=allUsers" className={classes.tabButton}>
+          Reviewed Users
+        </Link>
+        <div className={classNames(classes.tabButton, classes.recentlyActiveTab)}>
+          Recently Active Users <LoadMore {...recentlyActiveLoadMoreProps}/>
+        </div>
+      </div>
+      <table className={classes.table}>
+        <thead className={classes.header}>
+          <tr>
+            <td></td>
+            <td>Index</td>
+            <td>DisplayName</td>
+            <td className={classNames(classes.cell, classes.numberCell, {[classes.selected]: sorting === "karma"})} 
+              onClick={() => setSorting("karma")}>
+              Total Karma
+            </td>
+            <td className={classNames(classes.cell, classes.numberCell, {[classes.selected]: sorting === "last20Karma"})} 
+              onClick={() => setSorting("last20Karma")}>
+              Recent
+            </td>
+            <td className={classNames(classes.cell, classes.numberCell, {[classes.selected]: sorting === "lastMonthKarma"})} 
+              onClick={() => setSorting("lastMonthKarma")}>
+              Last Month
+            </td>
+            <td className={classNames(classes.cell, classes.numberCell, {[classes.selected]: sorting === "downvoters"})} 
+              onClick={() => setSorting("downvoters")}>
+              Downvoters
+            </td>
+            <td className={classNames(classes.cell, classes.numberCell, {[classes.selected]: sorting === "lastNotificationsCheck"})} 
+              onClick={() => setSorting("lastNotificationsCheck")}>
+              Last Checked
+            </td>
+            <td className={classNames(classes.cell, {[classes.selected]: sorting === "userSortByRateLimitCount"})} 
+              onClick={() => setSorting("userSortByRateLimitCount")}>
+              Rate Limits
+            </td>
+            <td></td>
+          </tr>
+        </thead>
+        <tbody>
+          {sortedUsers.map((user, i) => {
```

Comment:
Everything inside of this feels like it probably wants to be its own component, but doesn't have to be in this PR